backend/app/tests/ui/fixtures/ui.py

```diff
@@ -7,7 +7,7 @@ def chrome_page():
     """Return playwright chromium browser page"""
     playwright_sync = sync_playwright().start()
     chrome_browser = playwright_sync.chromium.launch(
-        headless=True, slow_mo=350, timeout=99999
+        headless=True, slow_mo=750, timeout=99999
```

Comment:
We should eventually try to move to getting these configured via command line arguments. The command line arguments can also provide a default. In other words, when you run things locally and you want to slow things down you pass in `--slow_mo 1000` to pytest but otherwise it always does them by a default value.

The same should apply to headless.

Reply:
I can work on this in a different PR (slowmo and headless)